packages/jsii-rosetta/lib/commands/coverage.ts

```diff
@@ -7,7 +7,7 @@ export async function checkCoverage(assemblyLocations: readonly string[]): Promi
   logging.info(`Loading ${assemblyLocations.length} assemblies`);
   const assemblies = await loadAssemblies(assemblyLocations, false);
 
-  const snippets = Array.from(allTypeScriptSnippets(assemblies, true));
+  const snippets = Array.from(await allTypeScriptSnippets(assemblies, true));
```

Comment:
`allTypeScriptSnippets` returns a promise to an array already...

```suggestion
  const snippets = await allTypeScriptSnippets(assemblies, true);
```